test/k8sT/manifests/netpol-cyclonus/install-cyclonus.yml

```diff
@@ -11,11 +11,15 @@ spec:
         - command:
             - ./cyclonus
             - generate
-            - --mode=upstream
+            - --exclude=
+            - --include=upstream-e2e
+            - --retries=3
             - --noisy=true
             - --ignore-loopback=true
             - --cleanup-namespaces=true
+            - --server-port=80
+            - --server-protocol=tcp
           name: cyclonus
           imagePullPolicy: IfNotPresent
-          image: mfenwick100/cyclonus:v0.2.0
+          image: mfenwick100/cyclonus:v0.4.7
```

Comment:
Can we please pin the image to its SHA, similar to what we do in #19026, for increased security against potential supply chains attack? That's also something we like to do in general for every docker image as per our security policy :)

Reply:
Pinned.